packages/component-library/draft/Kaizen/TitleBlock/NavigationButton.scss

```diff
@@ -27,6 +27,7 @@
 
   @include ca-media-tablet-and-up {
     color: $kz-color-wisteria-800;
+    padding-top: 17px;
```

Comment:
I might be nitpicking, but perhaps this should be in rems, in case people update their font size settings - the alignment would go out of whack. Also I'm curious about this value - did you visually align it? Why does only tablet and up need it?

I was always meaning to add a `px-to-rem` helper mixin in Kaizen - basically it's just the px value divided by root font size (which for most or all browsers 16px).

Reply:
I think this'd be easier to talk about via a Zoom chat where I can show you the visual.

Reply:
For posterity's sake:

> perhaps this should be in rems, in case people update their font size settings - the alignment would go out of whack.
You might be right, I'll switch it and play around with the font size settings.

> did you visually align it?
Yep. Also ran it past two designers and got the thumbs up.

> Why does only tablet and up need it?
Because only tablet and up shows the buttons at all. The mobile size hides them.

Reply:
Regarding px/rem, I made this mixin a while back, which is currently in use in one of our private repos:

```
@function px2rem($pixels, $base-font-size: 16) {
  @return ($pixels / $base-font-size) * 1rem;
}
```
The `* 1rem` is due to scss wierdness when units are converted.

Reply:
I imagine we would generally want to use this (or something like it) whenever we want sizes to scale with text, and only not use it when we explicitly don't (border widths for example)

Reply:
(The answer to the other two questions are 👍 )

Reply:
Changing base font sizes rather than simply zooming is a very obscure thing to do, but some people do do it (I remember looking into stats and it was less than 1%). In Firefox:
<img width="1111" alt="image" src="https://user-images.githubusercontent.com/702885/75213550-91e0b380-57de-11ea-97c8-0be5da776de8.png">

My hazy memory is that the zoom feature (probably needed more horsepower) was introduced into browsers a long time after the "change font size" feature.